install.ps1

```diff
@@ -3,7 +3,7 @@ $env:NVM_HOME=$NVM_HOME
 $env:NVM_PATH=$NVM_HOME
 $env:NVM_SYMLINK="$NVM_HOME\nodejs"
 $env:PATH=$env:PATH + ";$NVM_HOME"
-Invoke-WebRequest -Uri https://github.com/coreybutler/nvm-windows/releases/download/1.1.7/nvm-noinstall.zip -OutFile nvm.zip
+Invoke-WebRequest -Uri https://github.com/coreybutler/nvm-windows/releases/download/latest/nvm-noinstall.zip -OutFile nvm.zip
```

Comment:
Might be good to somehow limit to the latest v1 here, so if there is a v2, the action doesn't break over night. Probably not possible when pulling a release, though. Perhaps just use 1.1.10?

Reply:
Done.